### DISCUSSION TITLE
Colored table row variant

### DISCUSSION THREAD

**Discussion by jolamar** [indiana-university/rivet-source#429] [Feature and enhancement requests]
## Summary
Allow colored rows in tables to signal success, warning, info or danger.

This could be similar to how the box component has implemented it https://rivet.iu.edu/components/layout/box/#colored-box-row-variants

## Screenshot
<img width="1113" alt="screen shot 2019-02-19 at 12 03 29 pm" src="https://user-images.githubusercontent.com/1017119/53040344-6b0ad700-343e-11e9-8b22-d009f0b01cce.png">

**Comment by levimcg** [comment]
@jolamar I think this is a good idea, but I would be interested to see some research on how we make this pattern accessible, or even how we'd go about marking this up correctly. 

@mmeller75 Would you have any time to look into whether this is a common pattern and how we might make it accessible? Thanks 🙌

**Comment by mmeller75** [comment]
I haven't done any research as yet, but one consideration is that the color can't be the only signifier. Color as an enhancement may help some audiences, but (as I'm sure you know) relying on only color is a barrier for certain audiences.

**Comment by mmeller75** [comment]
I ran this by Greg just to make sure I wasn't missing anything. There shouldn't be a problem with doing this as long as color isn't the only indicator. 

The example screenshot actually does a good job of demonstrating this: even if you can't see the color (or the bold text), there are clearly other buttons indicating that something is different with that row. 

Some considerations: 
- It would be nice to have a way to quickly navigate to that row. If I need to do something, can I get there without navigating through the whole table?
- For someone pulling up a page with a possibly long table, while using a screen reader, how would they know there is an alert on a row without reading the entire table? Would there also be an alert at the top of the page or would this have an aria-live? If someone can look at the page and quickly see that something needs their attention, how do you make that equivalent for someone who can't see the page?
